spec/requests/api/v1/listings/create_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+RSpec.describe 'POST /api/v1/listings', type: :request do
+  let!(:landlord) { create(:user) }
```

Comment:
all users should be landlords and be able to rent in the same account

Reply:
true! in the test we made a user and called it landlord as we have a foreign key of landlord_id in listing data-base